### ISSUE TITLE
Simulation details GUI, mark empty stdout/stderr files

### ISSUE THREAD

**Issue by berland** [equinor/ert#996]
Suggest to have a way of displaying which of these files are empty. Empty files are not so interesting to open
![image](https://user-images.githubusercontent.com/306834/91822987-dbe89d00-ec38-11ea-870a-ab2e25269bf2.png)

**Comment by jondequinor** [comment]
Yeah, _OPEN_ is also ambiguous. I think _View_ is closer to the truth. Also, its appearance should match the fact that it is clickable (i.e. a button). This button should be disabled if the stream is empty.
